charmhelpers/contrib/network/ovs/__init__.py

```diff
@@ -134,6 +134,16 @@ def set_manager(manager):
                            'ssl:{}'.format(manager)])
 
 
+def set_value(table, arg1, arg2):
+    ''' Set the controller for the local openvswitch '''
```

Comment:
Please could we use fully formed docstrings.

e.g. in this case (using double ", not single ')

"""Sets a record column=value for a table in the ovs configuration.
    Please see (some manual or reference for what these arguments can be.

    :param table: <what this is>
    :type table: str
    :param arg1: <what it is, or reference where to find them.
    :type arg1: str (???)
    :param arg2: <what is it ...>
    :type arg2: str <....>
    :raises CalledProcessException: <why might ovs-vsctl fail?>
    """

Are 'arg1' and 'arg2' really good names?  I'm thinking `record` and `column_value` might be the closest.  However, it's really a very large API surface, so having this as a general command is probably not useful.

Also, `subprocess.check_call(..)` can raise `CalledProcessException` if the command goes wrong, so it that should also go in the docstring as above.